app/src/main/res/values/strings.xml

```diff
@@ -152,4 +152,11 @@
     <string name="event_more">More</string>
     <string name="help">Help</string>
     <string name="feedback">Feedback</string>
+    <string name="for_leads">For Leads</string>
+    <string name="leads_welcome">Congrats for being a GDG organizer!</string>
+    <string name="leads_resources">You should check the GDG resource at https://drive.google.com/drive/#folders/0B55wxScz_BJtWW9aUnk2LUlNdEk</string>

```

Comment:
Since user cannot copy the link here and cannot memorize it, I don't see any usefulness of the link in the detail text. 
I think we can make it like "click here to access GDG resources"